### PR TITLE
Containerize integration test

### PR DESCRIPTION
**Pull request by paradigm**
/cc stevekuznetsov csrwng

### CODE REVIEW

File: cluster/ci/origin/pipelines/integration/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+def library_url = "${env.RELEASE_LIBRARY_URL ?: 'github.com/openshift/release-library'}"
+def library_ref = "${env.RELEASE_LIBRARY_REF ?: 'master'}"
+
+library "${library_url}@${library_ref}"
+
+def templateBase = "cluster/ci/origin/config"
+
+pipeline {
+  agent { label "release-ci.jenkins-slave" }
```
Comment (csrwng) [inline cluster/ci/origin/pipelines/integration/Jenkinsfile]: agent any

File: cluster/ci/origin/pipelines/integration/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        waitForTag(this, this._buildName, "binaries", 7200)
+      }
+    }
+    stage("Integration Test") {
+      steps {
+        ensureLoggingComponents(this)
+        script {
+          def binaryImageRef = imageStreamTagRef(this, this._buildName, "binaries")
+          def pullSecretRef = dockerCfgSecret(this, "default")
+          def integrationRunId = "${this._buildName}-ut-${currentBuild.number}"
```
Comment (csrwng) [inline cluster/ci/origin/pipelines/integration/Jenkinsfile]: s/ut/int/

File: cluster/ci/origin/standalone-jobs.yaml [hunk 3]
```diff
@@ -136,6 +136,62 @@ objects:
 - apiVersion: v1
   kind: ConfigMap
   metadata:
+    name: it-origin-job
```
Comment (csrwng) [inline cluster/ci/origin/standalone-jobs.yaml]: s/it/int/

File: cluster/ci/origin/standalone-jobs.yaml [hunk 4]
```diff
@@ -136,6 +136,62 @@ objects:
...
+    annotations:
+      ci.openshift.io/jenkins-job: "true"
+  data:
+    job.yml: |-
+      - job:
+          name: ${JOB_PREFIX}-it-origin
```
Comment (csrwng) [inline cluster/ci/origin/standalone-jobs.yaml]: s/it/int/

File: cluster/ci/origin/pipelines/integration/Jenkinsfile [hunk 5]
```diff
@@ -8,7 +8,7 @@ library "${library_url}@${library_ref}"
 def templateBase = "cluster/ci/origin/config"
 
 pipeline {
-  agent { label "release-ci.jenkins-slave" }
+  agent any { label "release-ci.jenkins-slave" }
```
Comment (stevekuznetsov) [inline cluster/ci/origin/pipelines/integration/Jenkinsfile]: We don't need the `{ label }` stanza here, so:

```groovy
pipeline {
  agent any

// ...
```

**Comment by stevekuznetsov** [comment]
@csrwng we're seeing this pipeline work -- the tests run fine. 11 fail, some are trying to start the `kubelet` and failing to find `iptables` and other deps. Some are failing to start `etcd`. We can tackle that as a follow-up

**Comment by stevekuznetsov** [comment]
[(2) [CI] Deploy the Integration Test pipeline for Origin Pull Requests](https://trello.com/c/NKQdda6O/358-2-ci-deploy-the-integration-test-pipeline-for-origin-pull-requests)

**Pull request merged**